src/clang-c-frontend/clang_c_convert_literals.cpp

```diff
@@ -51,7 +51,7 @@ bool clang_c_convertert::convert_integer_literal(
   if(get_type(integer_literal.getType(), type))
     return true;
 
-  assert(type.is_unsignedbv() || type.is_signedbv());
+  assert(type.is_unsignedbv() || type.is_signedbv()); // for "_x=100", false || true. Same as "assert(sum>100)"
```

Comment:
Improve this comment.

Reply:
Done. 
Removed this comment as this annotation is redundant as the code is self-explanatory.